quicklogic/common/cmake/quicklogic_install.cmake

```diff
@@ -56,6 +56,11 @@ function(DEFINE_QL_TOOLCHAIN_TARGET)
   install(FILES ${VPR_CONFIG}
           DESTINATION share/symbiflow/scripts/${FAMILY})
 
+  # Example design to run through the flow
+  install(FILES ${symbiflow-arch-defs_SOURCE_DIR}/quicklogic/${FAMILY}/tests/counter_16bit/*.v
+	  DESTINATION share/symbiflow/tests/counter_16bit
+	   PERMISSIONS WORLD_READ OWNER_WRITE OWNER_READ GROUP_READ)
```

Comment:
I believe that CI should add some install tests (a bit like the [xc7 ones](https://github.com/SymbiFlow/symbiflow-arch-defs/tree/master/xc/xc7/tests/install_test)), so that we can verify that what ends up in the install package correctly works.

Reply:
I will add the support for in the followup PR. Here is the issue no : 
https://github.com/SymbiFlow/symbiflow-arch-defs/issues/2128